modules/ROOT/pages/notifications-configuration-reference.adoc

```diff
@@ -0,0 +1,113 @@
+= Notifications Configuration Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, notifications, error messages, responses, exceptions, monitor
+
+This topic provides details on the elements to configure for notifications.
+//DOCS-4183 For more information on notifications, see xref:mule-server-notifications.adoc[Mule Server Notifications].
+
+== Notifications
+
+Registers listeners for notifications, and associates interfaces with specific events.
+
+=== Attributes of Notifications
+
+[%header%autowidth.spread]
+|===
+|Name |Description
+|`dynamic` |If the notification manager is dynamic, you can set up the listeners to be registered dynamically at runtime. Otherwise, some parts of Mule will cache the notification configurations for efficiency reasons and will not generate events for newly-enabled notifications or listeners. +
+*Type*: `boolean` +
```

Comment:
I think **Type**, **Required** and **Default** should not be bold, since these are not UI elements. 
There are more occurrences of these items in the article.

Reply:
This is how it is throughout the doc and it would be a big job to change it. As we develop a style guide, we will probably have to back through a lot of topics or fix this kind of thing as we update.